Dockerfile-arm64

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 the Velero contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.13-buster AS build
+COPY . /go/src/velero-plugin-for-aws
+WORKDIR /go/src/velero-plugin-for-aws
+RUN GOARCH=arm64 CGO_ENABLED=0 GOOS=linux go build -v -o /go/bin/velero-plugin-for-aws ./velero-plugin-for-aws
+
+
+FROM arm64v8/ubuntu:bionic
```

Comment:
Can we use the more recent `ubuntu:focal` instead?

Reply:
This needs to be done on the other dockerfile too.
Going to clean this in a follow-up PR